examples/react-app/package.json

```diff
@@ -9,27 +9,31 @@
     "preview": "vite preview"
   },
   "dependencies": {
-    "@bonfida/sns-react": "file:../../react",
-    "@bonfida/spl-name-service": "file:../../js",
-    "@solana/spl-token": "^0.3.7",
-    "@solana/wallet-adapter-base": "^0.9.22",
-    "@solana/wallet-adapter-react": "^0.15.30",
-    "@solana/wallet-adapter-react-ui": "^0.9.29",
-    "@solana/wallet-adapter-wallets": "^0.19.15",
-    "@solana/web3.js": "^1.73.3",
+    "@bonfida/sns-react": "^2.0.2",
```

Comment:
I think it's better to define a "real" version of `@bonfida/sns-react` instead of just a path to the folder, because if we run it locally it will throw errors about different react instances/versions and we will anyway need to use `npm pack` to make it work properly. Also since it's an example, it should work with no additional hustle.

Reply:
I agree